app/controllers/tag_controller.rb

```diff
@@ -8,7 +8,8 @@ def index
 
     @title = I18n.t('tag_controller.tags')
     @paginated = true
-    @order_type = params[:order].blank? || params[:order] == "desc" ? "asc" : "desc"
+    @order_type = params[:order].blank? || (params[:order] == "desc") ? "desc" : "asc"
```

Comment:
Hmm, what's the difference between `order_type` and `order_name`? or should `order_name` be like, `name`, `followers` etc?

Reply:
Maybe we need more specific variable names here...

Reply:
`order_name` will be the order by  for 'names' so that we can default it to 'asc', while `order_type` will be used for 'uses' and  'followers for these we will default to 'desc' as we discussed...maybe i can use a better name `order_for_name` maybe?